pkgs/by-name/la/labwc-gtktheme/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, gobject-introspection
+, gtk3
+, python3Packages
+, wrapGAppsHook
+, unstableGitUpdater
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "labwc-gtktheme";
+  version = "unstable-2022-17-17";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "labwc";
+    repo = "labwc-gtktheme";
+    rev = "0eb103701775ecd3aa4d517f42dede4f56381241";
+    hash = "sha256-aeF6unzR9bqaKXkqOHlGrMdPx3xXCtig58tKVliUO4g=";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    gtk3
+  ];
+
+  pythonPath = with python3Packages; [
+    pygobject3
+  ];
+
+  strictDeps = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp -a labwc-gtktheme.py $out/bin/
```

Comment:
Why not remove the extension?

Reply:
Done.

Reply:
`meta.mainProgram` should also be changed

Reply:
Done!